tests/phpunit/unit-tests/Model/TestModelPDFMetaBoxFailure.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace GFPDF\Model;
+
+use GFPDF\Controller\Controller_PDF;
+use GFPDF\Helper\Helper_Url_Signer;
+use GFPDF\View\View_PDF;
+use WP_UnitTestCase;
+
+/**
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2020, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ */
+
+/**
+ * Class TestModelPDFMetaBoxFailure
+ *
+ * @package GFPDF\Model
+ *
+ * @group   pdf
+ */
+class TestModelPDFMetaBoxFailure extends WP_UnitTestCase {
+
+	/**
+	 * The WP Unit Test Set up function
+	 *
+	 * @since 4.0
+	 */
+	public function setUp() {
+		global $gfpdf;
+
+		/* run parent method */
+		parent::setUp();
+
+		/* Setup our test classes */
+		$this->model = new Model_PDF( $gfpdf->gform, $gfpdf->log, $gfpdf->options, $gfpdf->data, $gfpdf->misc, $gfpdf->notices, $gfpdf->templates, new Helper_Url_Signer() );
+		$this->view  = new View_PDF( [], $gfpdf->gform, $gfpdf->log, $gfpdf->options, $gfpdf->data, $gfpdf->misc, $gfpdf->templates );
+
+		$this->controller = new Controller_PDF( $this->model, $this->view, $gfpdf->gform, $gfpdf->log, $gfpdf->misc );
+		$this->controller->init();
+	}
+
+	/**
+	 * Check our PDF detail list is displaying correctly
+	 *
+	 * @since 4.0
+	 */
+	public function test_view_pdf_entry_detail() {
+
+		/* Setup some test data */
+		ob_start();
+		$this->model->view_pdf_entry_detail( [ 'entry' => [ 'form_id' => 0 ] ] );
+		$html = ob_get_clean();
+		$this->assertNotFalse( strpos( $html, 'No PDFs available for this entry.' ) );
```

Comment:
We can use `$this->assertStringContainsString()` here https://phpunit.readthedocs.io/en/9.2/assertions.html#assertstringcontainsstring